tests/helpers.py

```diff
@@ -625,8 +633,15 @@ def get_all_pvs():
     )
     return ocp_pv_obj.get()
 
+# Todo
+
+
+'''
+To revert the counts of tries and delay after bz1726266
```

Comment:
Please follow the guidelines for workarounds and add the issue number here

Reply:
Updated